src/helpers/hub.ts

```diff
@@ -1,9 +1,10 @@
 export type Listener = (type: string, payload: any) => void;
```

Comment:
`Listener` doesn't look like to be used anymore. I suggest removing it.

Reply:
`Listener` is still used here: https://github.com/datacamp/datacamp-light/blob/33d9487fd6cd66f33b045fff579718585b91067c/src/redux/exercise.ts#L41

Reply:
Why not move it then? Are they really coupled? It's not that obvious in `datacamp-light/src/redux/exercise.ts`